community/kernel/src/main/java/org/neo4j/kernel/configuration/ssl/LegacySslPolicyConfig.java

```diff
@@ -54,4 +59,15 @@
     public static final Setting<File> tls_key_file =
             derivedSetting( "unsupported.dbms.security.tls_key_file", certificates_directory,
                     certificates -> new File( certificates, "neo4j.key" ), PATH );
+
+    @Internal
+    @Description( "Default encryption protocol used for legacy SSl policy." )
+    static final Setting<List<String>> default_security_protocol =
+            setting( "unsupported.dbms.security.protocol", STRING_LIST, TLS_VERSION_DEFAULTS );
+
+    @Internal
+    @Description( "Default encryption protocol used for legacy SSl policy." )
```

Comment:
Not encryption protocol, but ciphers or cipher suites used for SSL/TLS (upper-case L). Since most of the things here say TLS I think it is better to stick to that.